src/ImageViewerCaptureTool.cpp

```diff
@@ -19,16 +19,19 @@ ImageViewerCaptureTool::ImageViewerCaptureTool(uint width, uint height) {
 }
 
 ImageViewerCaptureTool::ImageViewerCaptureTool(double fovY, double fovX, uint value, bool isHeight) {
-    double aspectRatio = fovX / fovY;
     uint width, height;
+    double fovYrad = fovY * M_PI / 180.0;
```

Comment:
I kept it because the method `_viewer->getCamera()->setProjectionMatrixAsPerspective` receives the angle in degrees.
What do you suggest (keep or change)?